che/main.py

```diff
@@ -21,6 +21,8 @@
 PLUGINS_FILE = "plugins.json"
 THRESHOLD = 0.45 # Top similarity to be considered a match with an existing plugin
 
+if not os.path.exists(DATA_DIR): os.mkdir(DATA_DIR)
```

Comment:
It's better to add the data/ directory in the filesystem. You could add a .gitkeep file in order to track it.

Also is a bad practice to check if the data/ directory exists in runtime. 

Ref: https://stackoverflow.com/a/7229996

Reply:
My bad, you are right.

Commited new changes.